o4/o4.py

```diff
@@ -1052,6 +1052,11 @@ def man(cmd):
 
 
 def o4_status(changelist, depot, check_all, quick):
+    '''
+    Checks for files that don't match Perforce.
+    Returns the number of files that are bad in any way,
```

Comment:
Returns the number of files, excluding files opened for edit/add/delete, that are not matching checksum.